src/CaptureClient/CaptureClient.cpp

```diff
@@ -92,7 +100,8 @@ std::vector<ApiFunction> FindApiFunctions(const orbit_client_data::ModuleManager
 }
 
 [[nodiscard]] orbit_grpc_protos::CaptureOptions ToGrpcCaptureOptions(
-    const ClientCaptureOptions& options, const orbit_client_data::ModuleManager& module_manager) {
+    const ClientCaptureOptions& options, const orbit_client_data::ModuleManager& module_manager,
+    const orbit_client_data::ProcessData& process) {
```

Comment:
nit. I would rename `process` -> `process_data` to give a better indication of the type without having to look at the signature. Also elsewhere in the PR.